apps/wesh/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.7"
+
+services:
+  web:
+    image: ibz0/wesh:v0.2.1
+    restart: on-failure
+    stop_grace_period: 1m
+    ports:
+      - 7681:7681
+    volumes:
```

Comment:
I notice you haven't added any volumes to persist data for the installed tools. Are they all ephemeral are do some of them expect data to be persisted? If they need data persisted they should be defined here.